code/game/machinery/cryo.dm

```diff
@@ -481,7 +541,7 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 	for (var/atom/movable/x in src.contents)
 		if((x in component_parts) || (x == src.beaker))
 			continue
-		x.forceMove(src.loc)
+		x.forceMove(get_step(loc, SOUTH))//to avoid PLAAAAANES issues with our cryo cell
```

Comment:
Cryo should probably use `get_step(loc, dir)` to determine where it spits out occupants and items so that the tile where it does so can be set via changing the pod's `dir` in mapping.